src/query/vacuum.c

```diff
@@ -1092,6 +1092,63 @@ xvacuum (THREAD_ENTRY * thread_p)
 #endif /* SA_MODE */
 }
 
+/*
+ * xvacuum_dump - Dump the contents of vacuum
+ *
+ * return: nothing
+ *
+ *   outfp(in): FILE stream where to dump the vacuum. If NULL is given,
+ *            it is dumped to stdout.
+ */
+void
+xvacuum_dump (THREAD_ENTRY * thread_p, FILE * outfp)
+{
+  LOG_PAGEID min_log_pageid = NULL_PAGEID;
+  int archive_number;
+
+  if (outfp == NULL)
+    {
+      outfp = stdout;
+    }
+
+  /* vacuum_boot () != NO_ERROR || prm_get_bool_value (PRM_ID_DISABLE_VACUUM) == true */
+  if (!vacuum_is_safe_to_remove_archives ())
+    {
+      return;
```

Comment:
If it don't be returned a message, I think, user will get empty message.
You need to return warning message.

Reply:
In this case, how about displaying the message 'vacuum did not boot properly' as the result of execution?
If you want normal error handling, error codes and messages must be returned through the protocol. This is not currently considered.

Reply:
It sounds good.

Reply:
The result of executing the vacuumdb utility with --dump option when the system parameter 'vacuum_disable' is set to yes.
--CS-mode
```
$ cubrid vacuumdb --CS-mode --dump demodb
VACUUM did not boot properly.
```
--SA-mode
```
$ cubrid vacuumdb --SA-mode --dump demodb
VACUUM did not boot properly.
```